docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md

```diff
@@ -0,0 +1,139 @@
+   export UAA_URL=$(btp --format json get services/binding --name ${CIS_INSTANCE_NAME}-binding | jq -r '.credentials.uaa.url')
+   export PROVISIONING_SERVICE_URL=$(btp --format json get services/binding --name ${CIS_INSTANCE_NAME}-binding | jq -r '.credentials.endpoints.provisioning_service_url')
+   ```
+
+3. Get the access token for the Provisioning Service API using the client credentials:
+
+   ```bash
+   TOKEN=$(curl -s -X POST "${UAA_URL}/oauth/token" -H "Content-Type: application/x-www-form-urlencoded" -u "${CLIENT_ID}:${CLIENT_SECRET}" --data-urlencode "grant_type=client_credentials" | jq -r '.access_token')
+   ```
+
+4. Check if Kyma runtime is available for provisioning:
+
+   ```bash
+   curl -s "$PROVISIONING_SERVICE_URL/provisioning/v1/availableEnvironments" -H "accept: application/json" -H "Authorization: bearer $TOKEN" | jq
+   ```
+
+5. Set the **ENVIRONMENT_TYPE** and **SERVICE_NAME** environment variables to const values `kyma` and `kymaruntime`, and provide values for the **NAME**, **REGION**, **PLAN**, and **USER_ID** environment variables:
+
+   ```bash
+   export ENVIRONMENT_TYPE="kyma"
+   export SERVICE_NAME="kymaruntime"
+   export NAME={RUNTIME_NAME}
+   export REGION={CLUSTER_REGION}
+   export PLAN={KYMA_RUNTIME_PLAN_NAME}
+   export USER_ID={USER_ID}
+   ```
+
+6. Provision the Kyma runtime and save the instance ID in **INSTANCE_ID** environment variable:
+
+   ```bash
+   INSTANCE_ID=$(curl -s -X POST "$PROVISIONING_SERVICE_URL/provisioning/v1/environments" -H "accept: application/json" -H "Authorization: bearer $TOKEN" -H "Content-Type: application/json" -d "{\"environmentType\":\"$ENVIRONMENT_TYPE\",\"parameters\":{\"name\":\"$NAME\",\"region\":\"$REGION\"},\"planName\":\"$PLAN\",\"serviceName\":\"$SERVICE_NAME\",\"user\":\"$USER_ID\"}" | jq -r '.id')
+   ```
+
+7. Optionally, set the **EXPIRATION_SECONDS** environment variable to the number of seconds (from 600 to 7200) after which a binding expires:
+
+   ```bash
+   export EXPIRATION_SECONDS={EXPIRATION_SECONDS}
+   ```
+
+8. After the provisioning is completed, create the binding to get the kubeconfig:
+
+   ```bash
+   [ -z "$EXPIRATION_SECONDS" ] && \
+   curl -s -X PUT "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN" | jq -r '.credentials.kubeconfig' > kubeconfig.yaml || \
+   curl -s -X PUT "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN" -H "Content-Type: application/json" -d "{\"parameters\":{\"expiration_seconds\":$EXPIRATION_SECONDS}}" | jq -r '.credentials.kubeconfig' > kubeconfig.yaml
+   ```
+
+9. To access the cluster through kubectl, set the **KUBECONFIG** environment variable to the path of the kubeconfig file:
+
+   ```bash
+   export KUBECONFIG=kubeconfig.yaml
+   ```
+
+10. Verify the connection to the cluster by running a kubectl command to get pods:
```

Comment:
Starting from point 10, the code blocks are not indented properly. I think you need to add one more space at the beginning of these lines.
![Screenshot 2024-11-05 at 16 00 37](https://github.com/user-attachments/assets/4132d4f2-8308-4780-895c-6b0ea3fea149)

Reply:
corrected.